vestido/vestido-storefront/components/landing-view/Carousel.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from "react"
+import Autoplay from "embla-carousel-autoplay"
+import image1 from '../../assets/carousel/slide-1.jpg'
+import image2 from '../../assets/carousel/slide-2.jpg'
+import image3 from '../../assets/carousel/slide-3.jpg'
+import {
+  Carousel,
+  CarouselContent,
+  CarouselItem,
+} from "libs/shadcn-ui/src/ui/carousel"
+import Image from "next/image"
+import { Button } from "@vestido-ecommerce/shadcn-ui/button"
+
+const slides = [
+  {
```

Comment:
Please type this `Array<LandingCarouselItem>`

Reply:
Done